react/features/chat/functions.js

```diff
@@ -1,5 +1,49 @@
 // @flow
 
+import escapeRegexp from 'escape-string-regexp';
+import aliases from 'react-emoji-render/data/aliases';
+import emojiAsciiAliases from 'react-emoji-render/data/asciiAliases';
+
+/**
+ * An ASCII emoticon regexp array to find and replace old-style ASCII
+ * emoticons (such as :O) to new Unicode representation, so then devices
+ * and browsers that support them can render these natively without
+ * a 3rd party component.
+ *
+ * NOTE: this is currently only used on mobile, but it can be used
+ * on web too once we drop support for browsers that don't support
+ * unicode emoji rendering.
+ */
+const EMOTICON_REGEXP_ARRAY = [];
+
+(function() {
+    for (const [ key, value ] of Object.entries(emojiAsciiAliases)) {
```

Comment:
a very similar replacement can be done for the (now ubiquitous) slack-style emoji: `:wave:`